src/OrchardCore.Themes/TheAdmin/wwwroot/css/TheAdmin.css

```diff
@@ -29037,7 +29033,7 @@ html[dir][data-theme=default] .disable.custom-control-input:checked ~ .custom-co
 }
 html[data-theme=default] .action-bar {
   top: 3.3rem;
-  z-index: 1000;
+  z-index: 999;
```

Comment:
This is the wrong place to make that change. It needs to be changed on the .scss file that defines that style. And I would be careful about changing that z-index on a global layout element.

Reply:
good catch! both css changes need to be done in the `.scss` file then run `npm install && gulp` to generate the new css.